internal/storage/postgreSQL/sensorSQL/mqtt.go

```diff
@@ -0,0 +1,142 @@
+package sensorSQL
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"github.com/google/uuid"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage"
+	"github.com/green-ecolution/green-ecolution-backend/internal/storage/postgreSQL/entities/sensor"
+)
+
+type SensorRepository struct {
+	db *sql.DB
+}
+
+func NewSensorRepository(db *sql.DB) *SensorRepository {
+	return &SensorRepository{db: db}
+}
+
+func (r *SensorRepository) Insert(ctx context.Context, data *sensor.MqttEntity) (*sensor.MqttEntity, error) {
+	query := `
+        INSERT INTO sensors (id, tree_id, data)
+        VALUES ($1, $2, $3)
+        ON CONFLICT (id) DO UPDATE SET
+            tree_id = EXCLUDED.tree_id,
+            data = EXCLUDED.data
+        RETURNING id
+    `
+	return r.upsertSensor(ctx, query, data)
+}
+
+func (r *SensorRepository) Get(ctx context.Context, id string) (*sensor.MqttEntity, error) {
```

Comment:
Return the domain entity
```suggestion
func (r *SensorRepository) Get(ctx context.Context, id string) (*sensor.MqttPayload, error) {
```